module.json

```diff
@@ -7,7 +7,8 @@
   "homepage": "https://github.com/ARMmbed/mbed-client",
   "license": "Apache-2.0",
   "dependencies": {
-    "mbed-client-c": "^2.0.0"
+    "mbed-client-c": "^2.0.0",
+    "mbed-trace": "ARMmbed/mbed-trace"
```

Comment:
Why is this added like this ? It should be added as yotta module and not as HEAD . Isn't that this module is now published ?